content/en/docs/tasks/observability/distributed-tracing/opentelemetry/index.md

```diff
@@ -0,0 +1,159 @@
+---
+title: OpenTelemetry
+description: Learn how to configure the proxies to send OpenTelemetry traces to a Collector.
+weight: 10
+keywords: [telemetry,tracing,opentelemetry,span,port-forwarding]
+aliases:
+ - /docs/tasks/telemetry/distributed-tracing/opentelemetry/
+owner: istio/wg-policies-and-telemetry-maintainers
+test: yes
+---
+
+After completing this task, you will understand how to have your application participate in tracing with [OpenTelemetry](https://www.opentelemetry.io/), regardless of the language, framework, or platform you use to build your application.
+
+This task uses the [Bookinfo](/docs/examples/bookinfo/) sample as the example application and the
+[OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) as the receiver of traces.
+
+To learn how Istio handles tracing, visit this task's [overview](../overview/).
+
+Istio can be configured to export [OpenTelemetry Protocol (OTLP)](https://opentelemetry.io/docs/specs/otel/protocol/)
+traces via gRPC.
+
+## Deploy the OpenTelemetry Collector
+
+{{< boilerplate start-otel-collector-service >}}
+
+## Installation
+
+All tracing options can be configured globally via `MeshConfig`.
+To simplify configuration, it is recommended to create a single YAML file
+which you can pass to the `istioctl install -f` command.
+
+### Exporting via gRPC
+
+In this example, traces will be exported via OTLP/gRPC to the OpenTelemetry Collector.
+The example also enables the [environment resource detector](https://opentelemetry.io/docs/languages/js/resources/#adding-resources-with-environment-variables). The environment detector adds attributes from the environment variable
+`OTEL_RESOURCE_ATTRIBUTES` to the exported OpenTelemetry resource.
+
+    {{< text syntax=bash snip_id=none >}}
+    $ cat <<EOF > ./tracing-grpc.yaml
+    apiVersion: install.istio.io/v1alpha1
+    kind: IstioOperator
+    spec:
+      meshConfig:
+        enableTracing: true
+        extensionProviders:
+        - name: otel-tracing
+          opentelemetry:
+            port: 4317
+            service: opentelemetry-collector.observability.svc.cluster.local
+            resource_detectors:
+              environment: {}
+    EOF
+    $ istioctl install -f ./tracing-grpc.yaml --skip-confirmation
+    $ kubectl label namespace default istio-injection=enabled
+    {{< /text >}}
+
+## Enable tracing for mesh via Telemetry API
+
+Enable tracing by applying the following configuration:
+
+    {{< text syntax=bash snip_id=enable_telemetry >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: telemetry.istio.io/v1alpha1
+    kind: Telemetry
+    metadata:
+      name: otel-demo
+    spec:
+      tracing:
+      - providers:
+        - name: otel-tracing
+        randomSamplingPercentage: 100
+        customTags:
+          "my-attribute":
+            literal:
+              value: "default-value"
+    EOF
+    {{< /text >}}
```

Comment:
```suggestion
{{< text syntax=bash snip_id=enable_telemetry >}}
$ kubectl apply -f - <<EOF
apiVersion: telemetry.istio.io/v1alpha1
kind: Telemetry
metadata:
  name: otel-demo
spec:
  tracing:
  - providers:
    - name: otel-tracing
    randomSamplingPercentage: 100
    customTags:
      "my-attribute":
        literal:
          value: "default-value"
EOF
{{< /text >}}
```

Reply:
Fixed in [47ace2c](https://github.com/istio/istio.io/pull/14566/commits/47ace2c3b5ce0a01b2b823ba8a75eb2519419f7a)